src/IIIFPresentation/API/Attributes/EtagCachingAttribute.cs

```diff
@@ -100,7 +99,7 @@ private static EntityTagHeaderValue GenerateETag(Stream stream, string path)
             new EntityTagHeaderValue('"' + hashString +
```

Comment:
Could this statement be made into one line?
Also - could string interpolation be used to build this string?